pkg/aks/client.go

```diff
@@ -3,16 +3,18 @@ package aks
 import (
 	"context"
 	"fmt"
+	"net/http"
+	"regexp"
 	"time"
 
 	"github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2020-11-01/containerservice"
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-11-01/subscriptions"
```

Comment:
Could the version of the azure sdk be unified here? 2020-11-01 for both.

Reply:
Unfortunately this version doesn't exist, we are using already `2019-11-01` next is completely new Azure SDK and migration will be done in another  card https://github.com/rancher/aks-operator/issues/287